### PR TITLE
Feature/ados 48 layout directory

### PR DESCRIPTION
**Pull request by marcos-wms**

### CODE REVIEW

**Comment by marcos-wms** [comment]
imagen

File: src/app/screens/Directory/styles.js [hunk 1]
```diff
@@ -1,7 +1,72 @@
...
+
+export const ContainerList = styled.View`
   justify-content: center;
 `;
+
+export const ButtonSubmit = styled.TouchableOpacity`
+  width: 40%;
+  font-size: 14;
+  border-radius: 20;
+  margin-left: auto;
```
Comment (usaqui-albert) [inline src/app/screens/Directory/styles.js:49]: `margin-horizontal: auto;`

File: src/app/themes/index.js.BASE.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// @flow
```
Comment (usaqui-albert) [inline src/app/themes/index.js.BASE.js:1]: What is this file? @marcos-wms

File: src/app/themes/index.js.LOCAL.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// @flow
```
Comment (usaqui-albert) [inline src/app/themes/index.js.LOCAL.js:1]: And this one?

File: src/app/screens/Directory/styles.js (hunk 1 above)
Comment (marcos-wms) [reply src/app/screens/Directory/styles.js:49]: when I place margin-horizontal: auto; it doesn't work so i added it separately.

imagen

File: src/app/themes/index.js.BASE.js (hunk 2 above)
Comment (marcos-wms) [reply src/app/themes/index.js.BASE.js:1]: sorry bro these files were generated by tortoise and I delete them

File: src/app/themes/index.js.LOCAL.js (hunk 3 above)
Comment (marcos-wms) [reply src/app/themes/index.js.LOCAL.js:1]: sorry bro these files were generated by tortoise and I delete them

File: src/app/themes/index.js.BASE.js (hunk 2 above)
Comment (marcos-wms) [reply src/app/themes/index.js.BASE.js:1]: sorry bro these files were generated by tortoise and I delete them

File: src/app/screens/Directory/styles.js (hunk 1 above)
Comment (usaqui-albert) [reply src/app/screens/Directory/styles.js:49]: Why the submit button "Add a new account" is not centered? @marcos-wms

File: src/app/screens/Directory/index.js [hunk 4]
```diff
@@ -1,11 +1,59 @@
...
+  ButtonSubmit,
+  TextButton,
+  ContainerList,
+  LinearGradientShadow,
+} from './styles';
 
-import { Container } from './styles';
+const list = [
+  {
+    title: 'My Cityback account',
```
Comment (usaqui-albert) [inline src/app/screens/Directory/index.js:19]: Typo, "Citybank" instead of "Cityback".

File: src/app/screens/Directory/styles.js (hunk 1 above)
Comment (usaqui-albert) [reply src/app/screens/Directory/styles.js:49]: Oh I see hehehe thanks @marcos-wms

**Pull request merged**